.github/workflows/deploy.yml

```
@@ -0,0 +1,102 @@
+        if: ${{ matrix.region == 'europe-west4' }}
+        uses: 'google-github-actions/deploy-cloudrun@v2'
+        with:
+          image: europe-docker.pkg.dev/ghost-activitypub/activitypub/migrations:${{ needs.build-test-push.outputs.migrations_docker_version }}
+          region: ${{ matrix.region }}
+          job: stg-${{ matrix.region }}-activitypub-migrations
+          flags: '--wait --execute-now --set-cloudsql-instances=ghost-activitypub:${{ matrix.region }}:stg-${{ matrix.region }}-activitypub-primary'
+
+      - name: "Deploy ActivityPub Queue to Cloud Run"
+        uses: 'google-github-actions/deploy-cloudrun@v2'
+        with:
+          image: europe-docker.pkg.dev/ghost-activitypub/activitypub/activitypub:${{ needs.build-test-push.outputs.activitypub_docker_version }}
```

actionlint (1.7.4)

property "build-test-push" is not defined in object type {}

(expression)